README.md

```diff
@@ -1,84 +1,55 @@
-# razorpay-php
+<p align="center"><img src="https://razorpay.com/images/logo.png"></p>
```

Comment:
As a new visitor to your readme It isn't clear from any of the angle rather looks awful compared to Razorpay's website, docs & everything else. Also adding a few lines of HTML in a MD file wont make us anti-national though :laughing: Still its you people who can make the final call so as you wish, I was just putting up my views as a new user of your API/Pkg and why I won't use it :)